doc/examples/features_detection/plot_haar.py

```diff
@@ -0,0 +1,70 @@
+"""
+============================
+Haar-like feature descriptor
+============================
+
+Haar-like feature descriptors are a set of simple features which was used in
```

Comment:
Haar-like features are simple digital image features that were used in the first ...